app/Http/Middleware/HasBankAccount.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+use \App\Tenant as Tenant;
+
+class HasBankAccount
+{
+		/**
+		 * Handle an incoming request.
+		 *
+		 * @param  \Illuminate\Http\Request  $request
+		 * @param  \Closure  $next
+		 * @return mixed
+		 */
+		public function handle($request, Closure $next)
+		{
+			$tenant = new Tenant();
+			$hasbankaccount = $tenant->has_bank_account();      
```

Comment:
minor but lets use under_scores for naming
```suggestion
			$has_bank_account = $tenant->has_bank_account();      
```

Reply:
dont think we need this if we pass in the check directly